colossalai/zero/init_ctx/init_context.py

```diff
@@ -236,7 +247,8 @@ def hijack_context_config(self, **kwargs):
 
 
 def no_shard_zero_context(is_replicated: bool = True):
```

Comment:
get_no_shard_zero_context is more appropriate

Reply:
The return type of it is Context Manager. 
The utility is "with no_shard_zero_context():"
I think it's appropriate.

Reply:
OK